priv/templates/nimble_template/.credo.exs

```diff
@@ -161,9 +161,9 @@
            order:
              ~w/shortdoc moduledoc behaviour use import alias require module_attribute defstruct callback/a
          ]},
-        {Credo.Check.Consistency.MultiAliasImportRequireUse, []},
+        {Credo.Check.Consistency.MultiAliasImportRequireUse, false},
```

Comment:
is there any reason to disable this @byhbt

Reply:
Because as our compass, we allow import multiple alias 

![Screen Shot 2021-12-23 at 10 51 39 AM](https://user-images.githubusercontent.com/948856/147185489-5c54e65b-5051-4c9c-82a0-8c82d95f35a7.png)

Reference: https://nimblehq.co/compass/development/code-conventions/elixir/#syntax

Reply:
That isn't correct @byhbt I believe we missed the other options (import, require, use) in our compass as we created that before we applied credo.

For me, better to follow credo/elixir convention @byhbt so let not disable it @byhbt

Reply:
but I do not fully understand you @byhbt what do you mean by `we allow import multiple alias` 🤔 ---> that is the default config for `MultiAliasImportRequireUse` 🤔, it works as expected, have you research what is the `MultiAliasImportRequireUse` rule in Credo before `disable` it

Reply:
Here is the example, when i tried to follow the compass convention for importing multiple aliases with the same parent namespace. 

![Screen Shot 2021-12-23 at 11 54 30 AM](https://user-images.githubusercontent.com/948856/147189966-eeb71d26-4d06-4af7-96af-593de3839e89.png)

Reference: https://hexdocs.pm/credo/Credo.Check.Consistency.MultiAliasImportRequireUse.html

If we don't disable it, then we cannot apply the compass convention for this case.

Reply:
so that means there is some other place that is using `multiple alias` right @byhbt should we adjust it instead of `disable the credo` 🤔

As I understand, by disabling this credo rule here, in the long term we will have a mix between `multiple aliases` and single alias in our codebase which won't be good.

Reply:
After discuss via Huddle we agree to keep the credo check for `MultiAliasImportRequireUse`. https://github.com/nimblehq/elixir-templates/pull/132/commits/089f5c7e5e6823c8940d5db2a531e34e7ff39db7